prow/config/jobs_test.go

```diff
@@ -129,37 +132,122 @@ func TestCommentBodyMatches(t *testing.T) {
 	}{
 		{
 			"org/repo",
-			"this is a random comment",
+			"/this-is-a-random-comment gce",
```

Comment:
aside from `/ok-to-test`, there are some tests for `/test all` too. for now, i will remove irrelevant tests for `/ok-to-test`.